pkg/registry/util_test.go

```diff
@@ -273,3 +275,282 @@ func TestGenerateOCICreatedAnnotations(t *testing.T) {
 	}
 
 }
+
+func TestIsOCI(t *testing.T) {
+
+	tests := []struct {
+		name    string
+		url     string
+		isValid bool
+	}{
+		{
+			name:    "Valid url",
```

Comment:
nit:
```suggestion
			name:    "Valid URI",
```

(A URL is a location you might send e.g. HTTP requests; a URI is just an identifier, which is true for OCI references)

Reply:
(and other places where we refer to OCI URIs as URLs)

Reply:
Thanks @gjenkins8 - Hopefully thats covered now - I've updated all wording to be consistent I believe now!